.dockerignore

```diff
@@ -0,0 +1,37 @@
+### Dotfiles
+.dockerignore
+.env
+.git
+.gitignore
+.github
+.vs
+.vscode
+
+### Docker
+docker-compose.yml
+docker-compose.*.yml
+docker/
+!docker/web-docker-entrypoint.sh
+
+### Docs
+docs
+
+### Mac
+.DS_Store
+
+### .NET
+bin/
+obj/
+dist/
+lib/
+*.user
+
+### Playwright
+test-results/
+playwright-report/
+playwright/.cache/
+zap-reports/
+.auth/
```

Comment:
Sara has mentioned that .dockerignore doesn't use relative paths, so since none of these are in our root directory these rules will all be ignored. To use relative we need to put **/ in front of each one
See https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/ for more details